sc2parse/plugins.py

```diff
@@ -572,8 +576,8 @@ def ArmyTracker(replay):
                 unit.died_at = replay.frames
 
             # This understates the army strength by rounding birth up and death down
-            firstMinuteInArmy = int(math.ceil(unit.finished_at/960.0))
-            lastMinuteInArmy = unit.died_at/960
+            firstMinuteInArmy = int(math.ceil(unit.finished_at/960.0/speed_multiplier))
+            lastMinuteInArmy = int(unit.died_at/960/speed_multiplier)
```

Comment:
to reduce cognitive load for the reader, how about adding parens to clarify order of computation?   whenever i see `a/b/c` i have to think about precedence rules and have to think in order to understand what it's doing and verify correctness.

instead of `replay.frames/960/speed_multiplier`, `(replay.frames/960)/speed_multiplier`
or maybe even better is `frames_per_game_minute = 960 / speed_multiplier;  replayMinutesCompleted  = replay.frames / frames_per_game_minute` ?